components/StoryCard.tsx

```diff
@@ -95,10 +95,49 @@ const StoryCard = ({ story, state, index, addFilter }: StoryCardParams): JSX.Ele
     }
   });
 
+  const [val, saveStoryType] = usePivotal(async ({ apiKey, projectId }) => {
+    if (story.story_type !== type) {
+      const newStory = await PivotalHandler.updateStory({
+        apiKey,
+        projectId,
+        storyId: story.id,
+        payload: { story_type: type },
+      });
+
+      dispatch(editStory(newStory));
+    }
+  });
+
+  useEffect(() => {
+    saveStoryType();
+  }, [type]);
+
   const openEstimationModal = (e): void => {
     e.stopPropagation();
     setIsModalVisible(true);
   };
+
+  const StoryTypeSelect = (): JSX.Element => {
+    return (
+      <Select
+        placeholder="Type"
+        value={type}
+        size="mini"
+        disabled={story.id === 'pending'}
+        onChange={value => {
+          setType(value);
+          saveStoryType();
+        }}
+        width="80px">
```

Comment:
Did you try resizing with styled components?

Reply:
Yes, I tried without success